dashboard/app/views/pd/regional_partner_contact/thanks.html.haml

```diff
@@ -17,7 +17,6 @@
     community if and when they become available.
 
   In the meanwhile, take a look
-  = link_to 'here for some ideas', 'https://code.org/educate/curriculum/3rd-party'
-  on how to bring computer science to your community, and take a look
   = link_to 'here for the K-12 Code.org curriculum options', courses_path
-  available for no cost to all schools
+  available for no cost to all schools, or check out our list of
+  = link_to '3rd party computer science resources.', 'https://code.org/educate/curriculum/3rd-party'
```

Comment:
nit; "3rd party" looks really funky to me outside of a url; can this be "third-party"?